pkgs/by-name/lu/luminous-ttv/package.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "luminous-ttv";
+  version = "0.5.7";
+
+  src = fetchFromGitHub {
+    owner = "AlyoshaVasilieva";
+    repo = pname;
+    rev = "v{version}";
+    hash = "sha256-uaq5uiSp4lci27BxtqjdtUeiJvXhWo25lfFE+dQys6Y=";
+  };
+
+  cargoHash = "sha256-jbtHxarRQ8gpCBc/HZWSnkzMrlMMltpknUBV1SGVq/I=";
+
+  meta = {
+    description = "A Rust server to retrieve and relay a playlist for Twitch livestreams/VODs.";
+    homepage = "https://github.com/AlyoshaVasilieva/{pname}";
+    downloadPage = "https://github.com/AlyoshaVasilieva/{pname}/releases/latest";
+    changelog = "https://github.com/AlyoshaVasilieva/{pname}/releases/tag/v{version}";
```

Comment:
```suggestion
    homepage = "https://github.com/AlyoshaVasilieva/${pname}";
    downloadPage = "https://github.com/AlyoshaVasilieva/${pname}/releases/latest";
    changelog = "https://github.com/AlyoshaVasilieva/${pname}/releases/tag/v${version}";
```

Reply:
Better: do not use "${pname}"

https://github.com/NixOS/nixpkgs/pull/278611